cmd/gaia/app/sim_test.go

```diff
@@ -238,27 +240,20 @@ func appStateRandomizedFn(r *rand.Rand, accs []simulation.Account, genesisTimest
 	stakingGenesis.Pool.NotBondedTokens = sdk.NewInt((amount * numAccs) + (numInitiallyBonded * amount))
 	stakingGenesis.Validators = validators
 	stakingGenesis.Delegations = delegations
+	modulesGenesis[staking.ModuleName] = cdc.MustMarshalJSON(stakingGenesis)
 
+	// TODO make use NewGenesisState
```

Comment:
Does this need an issue?

Reply:
it's a part of the existing ongoing issue to use `NewXxx` commands everywhere